libs/atqr/domain/src/lib/domain-spec/services/User.disl

```diff
@@ -0,0 +1,49 @@
+domain ATQR
+
+  context User
+    repository UserRepository
+
+      createUser: Contact
+        rules:
+          - Contact.email does not exist in UserRepository.users.email
+        create User and return
+
+      findUserById :userId
+        ⮐ User or null
+
+      findUserByEmail :email
+        ⮐ User or null
+
+      updateUser :user
```

Comment:
There should not be a generic "update user" method. What is being updated? The intention behind the update must be made clear in the method name.

You can look through the specs to find places where the user is updated and create specific methods for each place.

That being said, in this task, you can define only the basic methods for creating and finding and leave specific methods to be defined when the story which needs it is implemented

Reply:
It makes more sense now. I will resolve the comment